src/mesido/workflows/io/write_output.py

```diff
@@ -347,13 +352,25 @@ def _name_to_asset(name):
                             results[f"{asset.name}__variable_operational_cost"][0]
                             + results[f"{asset.name}__fixed_operational_cost"][0]
                         )
+                        asset_lifetime_opex_breakdown[asset.asset_type] += (
+                            results[f"{asset.name}__variable_operational_cost"][0]
+                            + results[f"{asset.name}__fixed_operational_cost"][0]
+                        ) * asset.attributes["technicalLifetime"]
```

Comment:
In MESIDO we already pass on the information of the technial lifetime, as sometimes a default is used (30years) I would prefer to extract the technical life from the MESIDO parameters. This also allows us to always compute the opex breakdown for the lifetime.
```suggestion
                        ) * parameters[f"{asset.name}.technical_life"]
```

Reply:
Agree. Updated as suggested